userspace/otpilot/Build.mk

```diff
@@ -15,3 +15,16 @@
 # SPDX-License-Identifier: Apache-2.0
 
 RUST_APPS_papa += otpilot
+
+# This is to work around a problem with calling git describe directly
+# from build.rs where it was always reporting dirty due to cargo
+# always rewriting the Cargo.lock of elf2tab .  We call git describe
+# here and leave the output in a file in the top directory, and then
+# build.rs will read this file.
+
+.PHONY: makegitlongtag
+
+makegitlongtag:
```

Comment:
Even though this is a `.PHONY` target, it still produces a file, so we should probably make the action's name match the file it generates.

Reply:
Yep.   New push to this PR does that.  (Still makes it .PHONY to make sure it happens every time.)